sonar-project.properties

```
@@ -0,0 +1,18 @@
+sonar.projectKey=WebUI-Project-Based-07-vk_BackEnd
+sonar.organization=webui-project-based-07-vk
+
+sonar.projectName=BackEnd
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+sonar.sources=.
+sonar.exclusions=**/src/tests/**/*, **/build-story/*
+sonar.coverage.exclusions=**/src/constants/*, **/src/tests/**/*, **/*.styles.ts
+
+sonar.tests=./src/tests
```

In this repo tests are placed in folder `test`.
```suggestion
sonar.tests=./src/test
```